functionary/ui/templates/partials/task_result.html

```diff
@@ -15,11 +15,7 @@
         <i id="result_indicator" class="fas fa-spinner fa-spin fa-2x"></i>
     {% elif task.result is None %}
         {% include "partials/output_none.html" %}
```

Comment:
No love for the None result? This could still have some logs to view.

Reply:
That logic was already there, although I just changed it to be `task.result == ""`, since that logic would never be true. I believe `task.result` will always just be an empty string since its a `TextField`. Now if you view a task that did not produce any output (like the `long_running_task`), you should see the partial get rendered

Reply:
Nevermind. This was because I was using old template images. Reverting back to the original logic